### DISCUSSION TITLE
Cannot extract base64 encoded images

### DISCUSSION THREAD

**Discussion by max-dw-i** [jgm/pandoc#10126] [Q&A] [answered]
There is an `.md` file with base64-encoded images in it: 
[test.md](https://github.com/user-attachments/files/16788584/test.md)

I'd like to extract these images into separate files. Here is the command I use:

```bash
pandoc --verbose --extract-media=./images test.md -o extracted.md
```

But only one image is created in the `images` folder (when there are 2 images in the file). Also this image has the size of 0B. Do I do anything wrong?

I'm on Fedora 40 at the moment. The `pandoc` version:

```
pandoc 3.1.3
Features: -server +lua
Scripting engine: Lua 5.4
```

**Comment by jgm** [comment, marked as answer]
I just tried this with latest pandoc (3.3) and it works fine.

**Reply by max-dw-i** [reply]
I'm an idiot. I thought 3.1 is the last version. Everything works with 3.3. Thank you.
